docs/install/index.md

```diff
@@ -0,0 +1,253 @@
+For these situations, Plone 6 may be installed manually.
+This method takes longer.
+It might be a challenge if you bump up against system requirements, or need to resolve conflicts between required packages.
+
+
+(install-index-image-installation-method-label)=
+
+## Image installation method
+
+The following Plone images are available.
+
+-   [`plone/plone-backend`](https://github.com/plone/plone-backend)
+-   [`plone/plone-frontend`](https://github.com/plone/plone-frontend) (Volto)
+-   [`plone/plone-haproxy`](https://github.com/plone/plone-haproxy)
+
+See their README files for installation and configuration instructions.
+
+
+(install-index-manual-installation-method-label)=
+
+## Manual installation method
+
+As an overview, you will perform the following steps in order.
+
+1.  Install the Plone backend (which includes the Classic UI) with either [buildout](install-index-install-backend-label) or [pip](install-index-install-backend-pip-label).
+1.  Create the Plone Site in a web browser, choosing either the new Volto or the Classic UI for a frontend.
+1.  Install the Plone frontend (Volto) with node.
+
+
+(install-index-install-backend-label)=
+
+### Install the Plone backend
+
+You can install the Plone backend with either [buildout](install-index-install-backend-label) or [pip](install-index-install-backend-pip-label).
+
+
+(install-index-install-backend-buildout-label)=
+
+#### Install backend with buildout
+
+Create a new directory to hold your project, make it your current directory, and create a new file inside it called `buildout.cfg`.
+This file will refer to the latest version of Plone, and the `user` value that will be used in the next step to create a Plone site.
+Paste the following configuration information into it.
+
+```ini
+[buildout]
+extends = https://dist.plone.org/release/6.0.0a4/versions.cfg
+parts = instance
+
+[instance]
+recipe = plone.recipe.zope2instance
+eggs =
+    Plone
+user = admin:admin
+```
+
+Install Plone with the following shell commands.
+
+```shell
+# Create a Python virtual environment in the current directory
+python3.9 -m venv .
+# Install the latest Plone 6 requirements with pip
+bin/pip install -r https://dist.plone.org/release/6.0.0a4/requirements.txt
+# Run buildout to install Plone 6
+bin/buildout
+# Start the Plone instance
+bin/instance fg
+```
+
+
+(install-index-install-backend-pip-label)=
+
+#### Install backend with pip
+
+Create a new directory to hold your project, make it your current directory, then issue the following commands in a shell session.
+
+```shell
+# Create a Python virtual environment in the current directory
+python3.9 -m venv .
+# Update Python package management tools
+bin/pip install -U pip setuptools wheel
+# Install Plone 6 with constrained requirements with pip
+bin/pip install Plone -c https://dist.plone.org/release/6.0.0a4/constraints.txt
+# Create a Plone 6 site with the given username and password in the current directory
+bin/mkwsgiinstance -u admin:admin -d .
```

Comment:
mkwsgiinstance is not enough, like no blobs configured. I would like to point to cookiecutter-zope-instance here.

Reply:
this one https://github.com/bluedynamics/cookiecutter-zope-instance/
I can move it to zopefoundation or plone repos if this feels better.

Reply:
Yeah, and the ensuing warning is... not a good look. I will trust you and other core developers to update this as you feel is best.

Reply:
Well, we have two sprints now, and I am at both. This is part of our installation story, so stay tuned. As said, mkwsgiinstance wont make anybody happy.

Reply:
FYI, I will also remotely participate in both upcoming sprints. I would be happy to act as a rubber duck.